main.go

```diff
@@ -131,16 +131,18 @@ func setPalette() {
 	if v == "" {
 		return
 	}
+
 	vals := strings.Split(v, ",")
-	if len(vals) != 2 {
-		return
-	}
-	if c, ok := colors[vals[0]]; ok {
-		fail = color.New(c)
-	}
-	if c, ok := colors[vals[1]]; ok {
-		success = color.New(c)
+	states := []*color.Color{fail, success, skipped}
+	for i := range vals {
+		if c, ok := colors[vals[i]]; ok {
+			states[i] = color.New(c)
+		}
 	}
+
+	fail = states[0]
+	success = states[1]
+	skipped = states[2]
```

Comment:
My initial thought was to dereference the `*color.Color` vars in `states` so that we can set their values directly on line 139, but I like this solution better. It's much clearer what's going on.

Reply:
For clarity, this is what I was thinking:
```
if c, ok := colors[vals[i]]; ok {
    *states[i] = *color.New(c)
}
```